api/envoy/config/bootstrap/v3/bootstrap.proto

```diff
@@ -642,3 +644,16 @@ message CustomInlineHeader {
   // The type of the header that is expected to be set as the inline header.
   InlineHeaderType inline_header_type = 2 [(validate.rules).enum = {defined_only: true}];
 }
+
+// Configuration for the Buffer Factories that create Buffers and Accounts.
+message BufferFactoryConfig {
```

Comment:
Looking into the feasibility of this in an overload_manager action:

`api_impl`  creates `watermark buffer factories` through its creation of `dispatchers`. The watermark buffer factories should know the minimum account sizes to track at creation.

If we shift the knowledge of the minimum account size to an overload manager action, then the `api_impl` needs access to the `overload manager` in order to create dispatchers. But the overload manager needs the api object in its ctor. 

Going down this route seems very convoluted given that dependence. 

We could have instead the `api_impl` parse some of the overload manager config from the bootstrap to get the knowledge, but that also seems wrong since the module is leaking.

As such I'm leaning towards wrapping this in `ResourceTrackingConfig` or similar as htuch recommends to future proof it.

Reply:
Done, moved to top level of the Overload Manager config.